src/data/scripts/huggingface_create/README.md

```diff
@@ -0,0 +1,40 @@
+# Hugging Face Dataset Creation Script
+This script is designed to create datasets for use with the Hugging Face Transformers library. It takes JSONL files containing data and converts them into a format compatible with the Hugging Face Datasets library. The resulting dataset can then be easily loaded and used for training and evaluation in various NLP tasks.
+
+## Usage
+This script is intended to be used for creating custom datasets that can be utilized with Hugging Face Transformers. It takes as input a set of JSONL files containing data and produces a Hugging Face dataset in the desired format.
+
+## Dependencies
+Before using this script, make sure you have the following dependencies installed:
+
+datasets
+pandas
+tqdm
+You can typically install these dependencies using pip:
+
+```bash
+pip install datasets pandas tqdm
```

Comment:
Can you lock the version for the library of this script? In the future, if datasets or pandas are updated to a breaking change version (X.x.x) and we don't have the correct version mentioned in the readme, this script will become obsolete.

Reply:
done